changes/add_GLT-4229.md

```diff
@@ -0,0 +1 @@
+Prepared Samples table on Edit Requisition Page to show all downstream samples
```

Comment:
This doesn't mention what happened to the Extractions table. Should be a `change` type and note that the Extractions table was changed to Prepared Samples. Try to include "(detailed sample)" at the end of any notes that only affect detailed sample mode too